src/components/molecules/Visualizer/Engine/ref.ts

```diff
@@ -64,6 +64,18 @@ export type EngineRef = {
   changeSceneMode: (sceneMode: SceneMode | undefined, duration?: number) => void;
   getClock: () => Clock | undefined;
   captureScreen: (type?: string, encoderOptions?: number) => string | undefined;
+  enableScreenSpaceCameraController: () => void;
+  disableScreenSpaceCameraController: () => void;
```

Comment:
```suggestion
  enableScreenSpaceCameraController: (enabled: boolean) => void;
```

Combining them is better to reduce duplicated codes